File: Graph/python/graph_adjMatrix.py
import numpy as np

def buildGraph(verticeCount):
    graph = np.zeros((verticeCount, verticeCount), dtype=int)
    return graph

def addVertice(graph, src, des, weight=1):
    graph[src][des] = weight

g1 = buildGraph(5)
print(g1)
addVertice(g1, 0, 1, 2)
addVertice(g1, 0, 3, 5)
addVertice(g1, 0, 4, 9)
addVertice(g1, 1, 2, 3)
addVertice(g1, 2, 3, 8)
addVertice(g1, 4, 3, 7)
print('*'*30)
print(g1)

File: Graph/java/graphAdjMatrix.java
public class graphAdjMatrix {
    public static void add_edge(int[][] graph, int src, int des, int wei){
        graph[src][des] = wei;
    }
    public static void main(String[] args) {
        int[][] graphMatrix = new int[5][5];
        add_edge(graphMatrix, 0, 1, 3);
        add_edge(graphMatrix, 0, 3, 8);
        add_edge(graphMatrix, 0, 4, 6);
        add_edge(graphMatrix, 1, 2, 5);
        add_edge(graphMatrix, 2, 3, 9);
        add_edge(graphMatrix, 3, 4, 7);
        
    }
}
